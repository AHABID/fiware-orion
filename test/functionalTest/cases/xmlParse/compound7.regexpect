7: +++++++++++++++++++++++++++++++++
HTTP/1.1 200 OK
Content-Length: 251
Content-Type: application/xml
Date: REGEX(.*)

<?xml version="1.0"?>
<updateContextResponse>
  <errorCode>
    <code>400</code>
    <reasonPhrase>Bad Request</reasonPhrase>
    <details>compound value error: bad tag-name of vector item: 'vecitem6' (should be 'vecitem')</details>
  </errorCode>
</updateContextResponse>
