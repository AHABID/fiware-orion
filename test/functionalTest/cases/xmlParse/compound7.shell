#   7. err: contextValue is a vector with five items + one item with a bad name
echo "7: +++++++++++++++++++++++++++++++++"
url='/ngsi10/updateContext'
payload='<?xml version="1.0"?>
<updateContextRequest>
  <contextElementList>
    <contextElement>
      <entityId type="Room" isPattern="false">
        <id>ConferenceRoom</id>
      </entityId>
      <contextAttributeList>
        <contextAttribute>
          <name>temperature</name>
          <type>degree</type>
          <contextValue type="vector">
            <vecitem>1</vecitem>
            <vecitem>2</vecitem>
            <vecitem>3</vecitem>
            <vecitem>4</vecitem>
            <vecitem>5</vecitem>
            <vecitem6>6</vecitem6>
          </contextValue>
        </contextAttribute>
      </contextAttributeList>
    </contextElement>
  </contextElementList>
  <updateAction>APPEND</updateAction>
</updateContextRequest>'
curlXml ${url} "${payload}"

