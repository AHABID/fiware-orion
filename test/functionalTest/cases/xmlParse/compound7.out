7: +++++++++++++++++++++++++++++++++
HTTP/1.1 200 OK
Content-Length: 251
Content-Type: application/xml
Date: Sun, 01 Jun 2014 17:45:20 GMT

<?xml version="1.0"?>
<updateContextResponse>
  <errorCode>
    <code>400</code>
    <reasonPhrase>Bad Request</reasonPhrase>
    <details>compound value error: bad tag-name of vector item: 'vecitem6' (should be 'vecitem')</details>
  </errorCode>
</updateContextResponse>
